lib/samples/set_reference_scale/set_reference_scale.dart

```diff
@@ -124,100 +124,140 @@ class _SetReferenceScaleState extends State<SetReferenceScale>
     setState(() => _ready = true);
   }
 
-  StatefulBuilder showSettings(BuildContext context) {
-    // Show the settings dialog.
-    return StatefulBuilder(builder: (context, setNewState) {
-      return Dialog(
-        child: Padding(
-          padding: const EdgeInsets.all(8),
-          child: SingleChildScrollView(
-            child: Column(
-              mainAxisAlignment: MainAxisAlignment.spaceAround,
-              children: [
-                Center(
-                  child: Text('Settings',
-                      style: Theme.of(context).textTheme.headlineMedium),
-                ),
-                Center(
-                  child: Text('Reference Scale',
-                      style: Theme.of(context).textTheme.titleMedium),
-                ),
-                // Add a dropdown button for setting a new reference scale.
-                DropdownButton(
-                  alignment: Alignment.center,
-                  // Set the selected scale
-                  value: _scale,
-                  icon: const Icon(
-                    Icons.arrow_drop_down,
-                    color: Colors.deepPurple,
-                  ),
-                  // Set the callback to update the selected scale.
-                  onChanged: (newScale) {
-                    setNewState(() {
-                      _scale = newScale!;
-                      _map.referenceScale = _scale;
-                    });
-                  },
-                  items: _referenceScaleList,
-                ),
-                Text('Apply Reference Scale to Layers',
-                    style: Theme.of(context).textTheme.titleMedium),
-
-                // Add a list of checkboxes for selecting feature layers that will honor the reference scale.
-                Column(
-                  children: [
-                    // Create a checkbox for each feature layer.
-                    for (final layer in _allFeatureLayers)
-                      CheckboxListTile(
-                        value: _selectedFeatureLayers.contains(layer),
-                        onChanged: (value) {
-                          setNewState(() {
-                            // Update the selected feature layers list.
-                            if (value ?? false) {
-                              _selectedFeatureLayers.add(layer);
-                            } else {
-                              _selectedFeatureLayers.remove(layer);
-                            }
+  Widget showSettings(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.fromLTRB(
+        20.0,
+        20.0,
+        20.0,
+        0,
```

Comment:
Instead of `0` at the bottom, elsewhere we have:
```dart
        max(
          20.0,
          View.of(context).viewPadding.bottom /
              View.of(context).devicePixelRatio,
        ),
```
This is unfortunately necessary for devices that do not have safe-area controls on the bottom, such as iPhones with hardware "home" buttons (e.g. iPhone SE 3rd gen).